cinema/serializers.py

```diff
@@ -68,13 +78,77 @@ class Meta:
             "movie_title",
             "cinema_hall_name",
             "cinema_hall_capacity",
+            "tickets_available"
```

Comment:
The `tickets_available` field is included in the `fields` list of `MovieSessionListSerializer`, but it is not defined in the serializer. Ensure that this field is properly defined or remove it from the `fields` list if it's not needed.